packages/data-validation/vite.config.ts

```diff
@@ -0,0 +1,47 @@
+import { resolve } from 'node:path';
+import { defineConfig } from 'vitest/config';
+import dts from 'vite-plugin-dts';
+
+import { name } from './package.json';
+
+const libName = name
```

Comment:
This branch must be checked out a long time ago... `vite.config.ts` has been updated to using a share library. Please update this file using other `vite.config.ts` as examples.